python/cuml/tests/test_strategies.py

```diff
@@ -119,7 +119,9 @@ def test_regression_datasets(dataset):
 
 
 @given(split_datasets(regression_datasets()))
-@settings(suppress_health_check=[HealthCheck.too_slow])
+@settings(
+    suppress_health_check=[HealthCheck.too_slow, HealthCheck.data_too_large]
```

Comment:
I ran into that same issue the other day... we might have to turn all of these off outside of "test development" profile or so.

Reply:
Yeah, I'm not sure exactly how best to handle that. If it's really a problem for all of our tests, we might need to revisit some of our data generation strategies. On the other hand, if it's not impacting CI times, it probably doesn't matter too much anyway.